mappings/net/minecraft/world/gen/feature/PatchFeatureConfig.mapping

```diff
@@ -0,0 +1,55 @@
+CLASS net/minecraft/class_4638 net/minecraft/world/gen/feature/PatchFeatureConfig
+	CLASS class_4639 Builder
+		FIELD field_21248 stateProvider Lnet/minecraft/class_4651;
+		FIELD field_21249 blockPlacer Lnet/minecraft/class_4629;
+		FIELD field_21250 whitelist Ljava/util/Set;
+		FIELD field_21251 blacklist Ljava/util/Set;
+		FIELD field_21252 tries I
+		FIELD field_21253 xSpread I
+		FIELD field_21254 ySpread I
+		FIELD field_21255 zSpread I
+		FIELD field_21256 canReplace Z
+		FIELD field_21257 projected Z
+		FIELD field_21258 byWater Z
+		METHOD <init> (Lnet/minecraft/class_4651;Lnet/minecraft/class_4629;)V
+			ARG 1 stateProvider
+			ARG 2 blockPlacer
+		METHOD method_23416 replacing ()Lnet/minecraft/class_4638$class_4639;
+		METHOD method_23417 tries (I)Lnet/minecraft/class_4638$class_4639;
+			ARG 1 tries
+		METHOD method_23418 whitelist (Ljava/util/Set;)Lnet/minecraft/class_4638$class_4639;
+			ARG 1 whitelist
+		METHOD method_23419 noProjection ()Lnet/minecraft/class_4638$class_4639;
+		METHOD method_23420 xSpread (I)Lnet/minecraft/class_4638$class_4639;
+			ARG 1 xSpread
+		METHOD method_23421 blacklist (Ljava/util/Set;)Lnet/minecraft/class_4638$class_4639;
+			ARG 1 blacklist
+		METHOD method_23422 needWater ()Lnet/minecraft/class_4638$class_4639;
+		METHOD method_23423 ySpread (I)Lnet/minecraft/class_4638$class_4639;
+			ARG 1 ySpread
+		METHOD method_23424 build ()Lnet/minecraft/class_4638;
+		METHOD method_23425 zSpread (I)Lnet/minecraft/class_4638$class_4639;
+			ARG 1 zSpread
+	FIELD field_21237 stateProvider Lnet/minecraft/class_4651;
+	FIELD field_21238 blockPlacer Lnet/minecraft/class_4629;
+	FIELD field_21239 whitelist Ljava/util/Set;
+	FIELD field_21240 blacklist Ljava/util/Set;
+	FIELD field_21241 tries I
+	FIELD field_21242 xSpread I
+	FIELD field_21243 ySpread I
+	FIELD field_21244 zSpread I
+	FIELD field_21245 canReplace Z
+	FIELD field_21246 projected Z
+	FIELD field_21247 byWater Z
+	METHOD <init> (Lnet/minecraft/class_4651;Lnet/minecraft/class_4629;Ljava/util/Set;Ljava/util/Set;IIIIZZZ)V
+		ARG 1 stateProvider
+		ARG 2 blockPlacer
+		ARG 3 whitelist
+		ARG 4 blacklist
+		ARG 5 tries
+		ARG 6 xSpread
+		ARG 7 ySpread
+		ARG 8 zSpread
```

Comment:
And here

Reply:
Good point. Shouldn't have sticked to data field names because data field names are actually very unintuitive